src/plugins/simpleini/simpleini.c

```diff
@@ -18,33 +18,48 @@
 #include <kdbease.h>
 #include <kdberrors.h>
 #include <kdblogger.h>
+#include <kdbprivate.h>    // elektraStrNDup
```

Comment:
Yes, I was wondering why `elektraStrDup` is in kdbhelpers.h and `elektraStrNDup` is in kdbprivate.h